config/ui-functions/mimic3-cache

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+CACHE_DIR=${CACHE_DIR:-/vx/ui/mimic-cache}
```

Comment:
While using speech-dispatcher and caching seems like the smaller change to make, I wonder whether you tried running mimic3 as an HTTP server that can be queried. If so, is it fast enough for real-time? I remember you saying the startup time was a big part of the problem.

Reply:
yep, that is also happening, as you see below. There are three things making things slow:

- loading the voice model
- synthesizing an utterance
- starting up the python mimic3 client

The first is addressed by running the server – I've confirmed that my setup doesn't work if I turn off the server

The second is address by pre-warming the cache

The third is addressed by putting the cache in *front* of mimic3.